index.js

```diff
@@ -0,0 +1,237 @@
+  const myModal = document.getElementById('myMOdal');
+  myModal.style.display = 'flex';
+  const modalBody = `<div class="modal-content">
+  <button id="closemodalX" class="close-modal close">&times</button>
+  <h3 class="project-title-modal">${projectInfo.name}</h3>
+  <ul class="card-tech-modal">
+      <li>${projectInfo.technologies[0]}</li>
+      <li>${projectInfo.technologies[1]}</li>
+      <li>${projectInfo.technologies[2]}</li>
+  </ul>
+  <img class="project-image modal-img" src="${projectInfo.image}" alt="${projectInfo.alt}">
+  <p class="project-description">${projectInfo.description}</p>
+  <div class="btns-modal">
+      <button class="btn-modal seeLive">See live
+          <img class="iconModal" src="./images/Union (8).png">
+      </button>
+      <button class="btn-modal seeSource">See source
+          <img class="iconModal" src="./images/redes sociales/Vector (8).png">
+      </button>
+  </div>
+  <div class="next-prev-proj">
+      <button class="prev-btn"> &larr; Previous project</button>
+      <button class="prev-btn">Next Project &rarr;</button>
+  </div>
+</div>`;
+  myModal.innerHTML = modalBody;
+};
+
+const gall = document.querySelector('.gallery');
+
+const closeModal = () => {
+  const myModal = document.getElementById('myMOdal');
+  myModal.style.display = 'none';
+};
+
+projectInfo.forEach((project, index) => {
+  const projectId = project.id;
+  const projectName = project.name;
+  const projectImage = project.image;
+  const projectTech = project.technologies;
+  const projectImgAlt = project.alt;
+
+  const soloArticle = document.createElement('article');
+  soloArticle.classList.add('card', `card${index + 2}`);
+
+  const cardContainer = `<img class="imgp" src="${projectImage}" alt="${projectImgAlt}" width="100%">
+<div class="pData">
+    <h3>${projectName}</h3>
+    <ul class="projectUl">
+        <li class="projectLi">${projectTech[0]}</li>
+        <li class="projectLi">${projectTech[1]}</li>
+        <li class="projectLi">${projectTech[2]}</li>
+    </ul>
+    <button class="see myBtn" id="btn${projectId}">
+        See this project &nbsp →
+    </button>
+</div>
+</article>`;
+  soloArticle.innerHTML = `${cardContainer} <div class="card card-back${index + 2}"></div>`;
+  gall.appendChild(soloArticle);
+  const button = document.getElementById(`btn${projectId}`);
+  button.addEventListener('click', () => {
+    showModal(project);
+    const closemodalX = document.getElementById('closemodalX');
+    closemodalX.addEventListener('click', () => {
+      closeModal();
+    });
+  });
+});
+
+// form validation
+
+const errorMessage = document.querySelector('.error-message');
+const form = document.getElementById('contact');
+const email = document.getElementById('mail');
+const emailRegex = /^[a-z0-9_.]+@[a-z0-9_.]+\.[a-z0-9_.]+$/;
+
+form.addEventListener('submit', (e) => {
+  if (!emailRegex.test(email.value)) {
+    e.preventDefault();
+    errorMessage.style.display = 'block';
+  }
+});
```

Comment:
- [ ] As specified in the project requirement kindly make sure that the `email` submitted by the user is in all `lowercase` letters. Currently, your project doesn't fulfill this requirement as users can still `submit` a form with `uppercase` letters in the `email`.  See below for more details:
<img width="1105" alt="Screenshot 2023-05-25 at 16 58 43" src="https://github.com/Bleu-Yves-Sopoude/mb-portfolio/assets/107192076/4d1b4c04-774f-493a-867b-c17ba7be0532">

- You can make use of the `toLowerCase()` inbuilt function to achieve this. Also kindly consider displaying an error `message` near the `submit` button informing them of the error. This will create a good user experience for the user to know why the form isn't submitted.